src/meta/ActiveHostsMan.cpp

```diff
@@ -146,7 +146,9 @@ void ActiveHostsMan::cleanExpiredHosts() {
         LOG(INFO) << "set " << data.size() << " expired hosts to offline in meta rocksdb";
         kvstore_->asyncMultiPut(kDefaultSpaceId, kDefaultPartId, std::move(data),
                                 [] (kvstore::ResultCode code) {
-            CHECK_EQ(code, kvstore::ResultCode::SUCCEEDED);
+            if (code != kvstore::ResultCode::SUCCEEDED) {
```

Comment:
what to do if some failed? just logging?

Reply:
Typically,  the failure is caused by leader change.  So we could do nothing here.